tests/backends/test_pyevm.py

```diff
@@ -11,6 +15,17 @@
     is_pyevm_available,
 )
 
+from eth_tester.utils.math_contract import (
+    _deploy_math,
+    _make_call_math_transaction,
+    _decode_math_result,
+)
+from eth_tester.utils.throws_contract import (
+    _deploy_throws,
+    _make_call_throws_transaction,
+    _decode_throws_result,
```

Comment:
All of these imports (and this diff in general) suggest that these are not used and can be removed?